src/Simplex/Messaging/Agent/Store/SQLite.hs

```diff
@@ -412,15 +429,27 @@ instance (MonadUnliftIO m, MonadError StoreError m) => MonadAgentStore SQLiteSto
         (rcvQId, _) <- getConnection st connAlias
         case rcvQId of
           Just qId -> updateRcvQueueStatus st qId status
-          Nothing -> throwError SEBadConn
+          Nothing -> throwError SEBadQueueDirection
       SND -> do
```

Comment:
That's another thing I don't like - it looks like a command, we could rename for example to DSend/DirSend/DirectionSend. No need to change in this PR